.github/workflows/reusable-workflow.yml

```diff
@@ -0,0 +1,290 @@
+# Reusable workflow to run the following jobs:
+#
+#   - multiplatform-tests
+#     - [ ubuntu-22.04 | ubuntu-20.04 | windows-2019 | windows-2022 ]
+#     - [ Debug | Release ]
+#     - execute tests in different versions of ubuntu with different build types
+#
+#   - asan
+#     - ubuntu-22.04
+#     - execute tests with ASAN flag
+#
+#   - tsan
+#     - ubuntu-22.04
+#     - execute tests with TSAN flag
+#
+#   - clang
+#     - ubuntu-22.04
+#     - execute clang-tidy check
+#
+#   - coverage
+#     - ubuntu-22.04
+#     - execute test with coverage flag and upload results
+#
+#   - flaky
+#     - ubuntu-22.04
+#     - execute flaky tests
+#
+#   - docs
+#     - ubuntu-22.04
+#     - execute docs compile and tests
+#
+#   - uncrustify
+#     - ubuntu-22.04
+#     - test uncrustify
+#
+#   - python-linter
+#     - ubuntu-22.04
+#     - test python linter
+#
+
+name: test
+
+on:
+
+  workflow_call:
+    inputs:
+
+      custom_version_build:
+        description: >
+          Version of Fast DDS build from eProsima-CI.
+        required: true
+        type: string
```

Comment:
```suggestion
        type: string
        default: 'v2'
```

Reply:
I prefer to keep it blank and make it required.